README.md

```diff
@@ -230,12 +230,21 @@ curl -F "data=@<Path to first evaluation File>" -F "data=@<Path to second evalua
 
 ### How to Install
 
-If you would like to change Heimdall to your needs, Heimdall has 'Development Mode' you can use, where if you make changes to the code, the app will automattically rebuild itself and use those changes. Please note that you should *not* run development mode when deploying Heimdall for general usage. To get started on a Debian-based distribution, follow these steps:
+If you would like to change Heimdall to your needs, Heimdall has 'Development Mode' you can use, where if you make changes to the code, the app will automatically rebuild itself and use those changes. Please note that you should *not* run development mode when deploying Heimdall for general usage. To get started on a Debian-based distribution, follow these steps:
 
-1. Install system dependencies:
+1. Install system dependencies with your system's package manager.
 
+   Debian/Ubuntu:
+
+   - ```bash
+     sudo apt install postgresql nodejs=16 nano git
+     sudo npm install -g yarn
+     ```
+     
+   OSX:
+   
    - ```bash
-     sudo apt install postgresql nodejs nano git
+     brew install postgresql nodejs@16 nano git
```

Comment:
thoughts @wdower?

Reply:
I'd gone with 16 because I believe that's what @camdenmoors is using. I'd stick with that over 14 just because it's a later version, but agreed that we should eventually address these dependencies so that they're just the latest versions.